forwarder.go

```diff
@@ -32,6 +32,12 @@ func forwarder(c echo.Context, client *http.Client) error {
 	req := c.Request()
 	ctx := req.Context()
 
+	isAuthHeaderCheckEnabled := viper.GetBool("authHeaderCheck.enable")
```

Comment:
can you read it once and use it further?

Reply:
Done.